api/client/admin/options.go

```diff
@@ -0,0 +1,23 @@
+package admin
+
+var (
+	defaultCfgGetMiners = cfgGetMiners{
+		withPower: true,
+	}
+)
+
+type cfgGetMiners struct {
+	withPower bool
+}
+
+// GetMinersOption configures filters for getting miners.
+type GetMinersOption func(*cfgGetMiners) error
+
+// WithPowerGreaterThanZero filters miners that have power greater
+// than zero.
+func WithPowerGreaterThanZero(withPower bool) GetMinersOption {
+	return func(cfg *cfgGetMiners) error {
+		cfg.withPower = withPower
+		return nil
+	}
+}
```

Comment:
Uhm, not sure about that since this way allows to set default values. For example, in this case, the default value is `true`. If the API receives a pointer to a struct, then we can't control the default value of options.

Reply:
Ah didn't realize the default was `true`. Yea, better this way then.

Reply:
Well, maybe I take that back. You can just create a default gRPC request with the value set to `true`. If some provided option overrides that, then it is set again. Anyway like I said, not something that has to change, just interesting discussion about patterns.

Reply:
Sure, it's worth the discussion :). Yeah, the good thing about default values at the gRPC level is that other clients (say, JS or Python) will have default value too; as it doesn't depend on the client-implementer as here.